core/src/main/resources/i18n/displayStrings_cs.properties

```diff
@@ -3519,6 +3519,7 @@ validation.phone.missingCountryCode=K ověření telefonního čísla je potřeb
 validation.phone.invalidCharacters=Telefonní číslo {0} obsahuje neplatné znaky
 validation.phone.insufficientDigits=V čísle {0} není dostatek číslic, aby mohlo být platné telefonní číslo
 validation.phone.tooManyDigits=V čísle {0} je příliš mnoho číslic, než aby mohlo být platné telefonní číslo
+validation.phone.incorrectLength=The field must contain {0} numbers
```

Comment:
No need to add i18n strings to the non-default file. If not present it falls back to the default anyway